tests/acceptance/features/apiOcm/ocm.feature

```diff
@@ -0,0 +1,21 @@
+Feature: an user shares resources usin ScienceMesh application
+  As a user
+  I want to share resources between different ocis instances
+
+
+  Background:
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+    And using server "REMOTE"
+    And user "Brian" has been created with default attributes and without skeleton files
+
+
+  Scenario: user generates invitation 
+    Given using server "LOCAL"
+    When "Alice" generates invitation
```

Comment:
can invitation be created without specifying any recipient?

Reply:
Yes, here we are just creating a token. We can specify the user's email, but this is not a required field.